lib/dao.js

```diff
@@ -1876,8 +1911,13 @@ DataAccessObject.prototype.save = function (options, cb) {
   var hookState = {};
 
   if (options.validate === undefined) {
-    options.validate = true;
+    if (Model.settings.automaticValidation === undefined) {
+      options.validate = true;
+    } else {
+      options.validate = Model.settings.automaticValidation;
+    }
```

Comment:
Now that I'm trying: the old code also modified the options-object. When I use a local variable, some tests fail, because they expect options==true when it's undefined